app/src/main/java/com/codefororlando/orlandowalkingtours/PlaceholderFragment.kt

```diff
@@ -0,0 +1,18 @@
+package com.codefororlando.orlandowalkingtours
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+/**
+ * Created by ryan on 7/29/17.
+ */
+class PlaceholderFragment : Fragment() {
+    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view: View = inflater?.inflate(R.layout.fragment_placeholder, container, false)!!
```

Comment:
Using the !! operator is typically a code smell.  If the inflate ever returns null, you'll get an NPE here.